frontend/src/metabase/lib/i18n.js

```diff
@@ -26,8 +27,8 @@ export async function loadLocalization(locale) {
   setLocalization(translationsObject);
 }
 
-// Tell Moment.js to use the value of the start-of-week Setting for its current locale
-export function updateMomentStartOfWeek() {
+// Tell moment.js and day.js to use the value of the start-of-week Setting for its current locale
+export function updateMomentAndDayjsStartOfWeek() {
```

Comment:
nit: I'd have two separate functions: `updateMoment*` and `updateDayjs*` and then a common `updateStartOfWeek` which contains both those functions. So upcoming removal of moment will be less painful